# Description: This pacth adds a missing find_package for GObject.
#  Without this patch the package will FTBFS with binutils-gold.
# Forwarded: http://sourceforge.net/tracker/?func=detail&aid=2897376&group_id=208427&atid=1005654
# Author: José Manuel Santamaría Lema <panfaust@gmail.com>
# 
--- subtitlecomposer-0.5.3.orig/src/player/gstreamer/CMakeLists.txt
+++ subtitlecomposer-0.5.3/src/player/gstreamer/CMakeLists.txt
@@ -1,6 +1,7 @@
 # TODO WHY AREN'T THESE INCLUDED BY MACRO_OPTIONAL_FIND_PACKAGE( GStreamer )?
 FIND_PACKAGE( GLIB2 REQUIRED )
 FIND_PACKAGE( LibXml2 REQUIRED )
+FIND_PACKAGE( GObject REQUIRED )
 
 SET( players_gstreamer_INCLUDE_DIR
 	${GLIB2_INCLUDE_DIR}
